docs/content/contributing/releasing.md

```diff
@@ -42,7 +42,6 @@ If you are releasing a new minor release, then update the `metadata.yaml`s so th
 for e.g. local development and e2e tests:
 
 1. Add the new release to the root level `metadata.yaml` release series.
-1. Add the new release to the e2e configuration `test/e2e/data/shared/v1beta1-caren/metadata.yaml` release series.
```

Comment:
wrong instruction or removed by accident?

Reply:
Wrong instruction, it would already be there by following the next line on release.